doc/basic_configuration.rst

```diff
@@ -55,17 +59,34 @@ In *all* build files::
     emails:
     - your_email@example.com
 
+You need to have a `local smtp service configured <https://github.com/ros-infrastructure/buildfarm_deployment#setup-master-for-email-delivery>`_ for email notifications.
```

Comment:
This note lacks any context that it is meant for the Jenkins master.

Reply:
Thanks. Added.